runtime/amplitude/src/zenlink.rs

```diff
@@ -0,0 +1,183 @@
+use core::convert::TryInto;
+
+use super::*;
+
+use orml_traits::MultiCurrency;
+use sp_runtime::{DispatchError, DispatchResult};
+use sp_std::marker::PhantomData;
+
+use zenlink_protocol::*;
+pub type ZenlinkAssetId = zenlink_protocol::AssetId;
+
+parameter_types! {
+	pub SelfParaId: u32 = ParachainInfo::parachain_id().into();
+	pub const ZenlinkPalletId: PalletId = PalletId(*b"/zenlink");
+	pub ZenlinkRegisteredParaChains: Vec<(MultiLocation, u128)> = vec![
+		// (make_x2_location(2001), 10_000_000_000),
```

Comment:
Would it be important to leave this comment? Otherwise please remove it. Smae for the other `zenlink.rs` file.